configuration/uncontrast_initial_data_types.py

```diff
@@ -0,0 +1,196 @@
+main_paradigms = [
+    "Priming",
+    "Emergence to awareness",
+    "Attention allocation",
+    "Change Blindness",
+    "Learning",
+    "Cognitive Tasks",
+    "Computational modeling",
+    "Illusions",
+    "Sperling Like",
+    "Visual search"
+]
+
+specific_paradigms = {
```

Comment:
1. I think we should call this also "uncontrast paradigms|' etc like in the model (for text search in the IDE
2. I'd rather don't have a list and a dict which are related by "name" only but just a dict of specific paradigms, where main paradigm that don't have specific have an empty list, and iterate on the dict keys if you need the main paradigms